index.php

```diff
@@ -1,22 +1,192 @@
-<?php get_header(); ?>
-
-<div class="container">
-    <main id="main-content">
+<?php
+/*
+Template Name: Portfolio Page
+*/
+get_header(); ?>
+<main>
+    <section class="portfolio">
+        <img class="portfolio__image"
+            src="<?php echo get_template_directory_uri() . '/image/ezequielBackground.jpg'; ?>"
+            alt="Ezequiel Background"
+        />
+    </section>
+    <section class="portfolio__section">
+        <h1 class="portfolio__Iam">
+            <span><?php echo esc_html__('Soy', 'aprendiendoezequiel'); ?></span> 
+            <span class="portfolio__author"><?php echo esc_html(get_theme_mod('portfolio_name', 'EZEQUIEL BLANCO')); ?></span>
+        </h1>
         <?php
-        if (have_posts()) :
-            while (have_posts()) : the_post();
-                ?>
-                <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
-                    <h2><a href="<?php the_permalink(); ?>"><?php the_title(); ?></a></h2>
-                    <?php the_content(); ?>
-                </article>
-                <?php
-            endwhile;
-        else :
-            echo '<p>No hay contenido para mostrar.</p>';
-        endif;
+        $titles = get_theme_mod('portfolio_titles', array('Diseñador UI/UX', 'Pintor(retratos)', 'Front-end developer'));
+        if (!empty($titles)) : ?>
+            <ul class="portfolio__professions">
+                <?php foreach ($titles as $title) : ?>
+                    <li><?php echo esc_html($title); ?></li>
+                <?php endforeach; ?>
+            </ul>
+        <?php endif; ?>
+        <p class="portfolio__description ">
+            <?php echo esc_html(get_theme_mod('portfolio_description', 'Este es mi portfolio y CV hecho en Figma, acá puedes ver mi estilo de prototipado a la hora de diseñar interfaces.')); ?>
+        </p>
+        <p class="portfolio__greeting">
+            <?php echo esc_html(get_theme_mod('portfolio_greeting', '¡Espero te guste!')); ?>
+        </p>
+    </section>
+    <?php
+        $cv_obtaining_id = attachment_url_to_postid(wp_get_upload_dir()['baseurl'] . '/portafolio/CVEZE.pdf');
+        $cv_url = wp_get_attachment_url($cv_obtaining_id);
         ?>
-    </main>
-</div>
+        <section class="portfolio__btn">
```

Comment:
De aqui para abajo esta pasado de tabulacion
```suggestion
    <section class="portfolio__btn">
```